### PR TITLE
News - Add legacy navigation colours.

### PR DESCRIPTION
**Pull request by bigmartyn**
Adds colours needed by the News Navigation.

### CODE REVIEW

File: settings/_news-colours.scss [hunk 1]
```diff
@@ -17,6 +17,11 @@ $nw-c-mist: #f2f2f2;
 // link colours
 $nw-c-dark-blue: #1167a8;
 
+// navigation colours
+$nw-c-trueblack #000000
```
Comment (shaunbent) [inline settings/_news-colours.scss]: Missing colons and semi-colons after the variable name and at the end of each line.

**Comment by sareh** [comment]
:+1: Just needs update to the changelog.md :)

**Comment by sareh** [comment]
:+1:

**Comment by sareh** [comment]
👍 Looks good!

**Pull request merged**